Source/WebKit/NetworkProcess/storage/LocalStorageManager.h

```diff
@@ -66,12 +66,15 @@ class LocalStorageManager {
     void disconnectFromStorageArea(IPC::Connection::UniqueID, StorageAreaIdentifier);
 
     HashMap<String, String> fetchStorageMap() const;
+    void setLocalStorageMap(HashMap<String, String>, const WebCore::ClientOrigin&, Ref<WorkQueue>&&);
+    void setTransientLocalStorageMap(HashMap<String, String>, const WebCore::ClientOrigin&);
```

Comment:
HashMap<String, String>&& ?

Reply:
(Or, if WTFMove semantics don't work for some reason, at the very least `const HashMap<String, String>&`

Reply:
Fixed!